tests/LINEBot/EventRequestParserTest.php

```diff
@@ -549,6 +549,30 @@ class EventRequestParserTest extends TestCase
     "trackingId": "track_id"
    },
    "replyToken":"replytoken"
+  },
+  {
+   "replyToken": "nHuyWiB7yP5Zw52FIkcQobQuGDXCTA",
+   "type": "message",
+   "mode": "active",
+   "timestamp": 1462629479859,
+   "source": {
+    "type": "user",
+    "userId": "U4af4980629..."
+   },
+   "message": {
+    "id": "325708",
+    "type": "text",
+    "text": "@example Hello, world! (love)",
+    "mention": {
+     "mentionees": [
+      {
+       "index": 0,
+       "length": 8,
+       "userId": "U850014438e..."
```

Comment:
Please add mention message without userId data for code coverage.
And this userId should be full length

Reply:
OK, updated **userId** of testing mention.
And, added `testing without mentionee's userId` and `testing without mention`.